### PR TITLE
increase the dns timeout to a multiple of dns servers used

### PR DESCRIPTION
**Pull request by br1-msw**
All UTs pass fine. It's a simple change so I've not run FVs over this too.

### CODE REVIEW

**Comment by rkd-msw** [comment]
RND is right.

**Comment by rkd-msw** [comment]
RND is right.

**Comment by br1-msw** [comment]
Oops. Is that right though? In your email you've said 2 servers should be 200ms (not 300ms)

**Comment by RobDover** [comment]
I believe that was a typo - there is a later email in the trail where @rkd-msw corrects himself.

(See the email sent 18 January 2018 21:22)

**Comment by br1-msw** [comment]
Ah yes I hadn't spotted that email. - @RobDover I've now corrected the code to give the right values. Could I get a re-review?

File: include/dnscachedresolver.h [hunk 1]
```diff
@@ -83,12 +83,15 @@ class DnsCachedResolver
   // Reads DNS records from _dns_config_file and stores them in _static_records
   void reload_static_records();
 
-  // Default timeout for DNS requests over the wire (in milliseconds)
-  static const int DEFAULT_TIMEOUT = 200;
+  // Default timeout for DNS requests over the wire (in milliseconds) for a single DNS server
```
Comment (RobDover) [inline include/dnscachedresolver.h]: While this is correct, I don't think it encompasses the full meaning of this field any more. The field now represents the total timeout we allow across all DNS requests for a given query, and it's probably worth updating the comment here to make that clear.

**Pull request merged**